README.md

```diff
@@ -31,8 +35,6 @@ Build status (master): [![CircleCI](https://circleci.com/gh/stratum/stratum/tree
 
 [NP4 Intel](stratum/hal/bin/np4intel/docker/README.md)
 
-[Stratum inside Docker](TODO)
```

Comment:
This topic got covered by the READMEs for the individual platforms over time, in case anyone wonders.